docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -9,7 +9,8 @@ Internal authentication
 -----------------------
 
 Requests between Trino nodes are authenticated using a shared secret. For secure
-internal communication, the shared secret must be configured on all nodes in the cluster:
```

Comment:
Maybe keep the original, but say “must be set to the same value on all nodes”

Reply:
Done